asmcli/asmcli.sh

```diff
@@ -1272,8 +1322,13 @@ set_up_local_workspace() {
   fi
   pushd "$OUTPUT_DIR" > /dev/null
 
-  KUBECONFIG="asm_kubeconfig"
-  export KUBECONFIG
+  # If KUBECONFIG file is supplied, keep using that.
+  if [[ "${KUBECONFIG_SUPPLIED}" -eq 0 ]]; then
```

Comment:
Is there any way or need to validate the `KUBECONFIG` provided by the users is valid or existing? Or we should just use that until we encounter error?

Reply:
if the file doesn't exist, or if it exists but doesn't contain the proper content, the "testing cluster connectivity" check should capture it down the flow. does that suffice?

Reply:
Yes!